template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java

```diff
@@ -56,11 +57,12 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
         String cdhVersion = getCdhVersion(source);
         boolean is727OrNewer = isVersionNewerOrEqualThanLimited(cdhVersion, CLOUDERAMANAGER_VERSION_7_2_7);
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
-        boolean sdxHbaseCloudStorageEnabled =
-                entitlementService.sdxHbaseCloudStorageEnabled(accountId);
+        boolean sdxHbaseCloudStorageEnabled = entitlementService.sdxHbaseCloudStorageEnabled(accountId);
+        boolean razEnabled = entitlementService.razEnabled(accountId);
```

Comment:
This only checks if the entitlement is enabled. The entitlement being enabled doesn't mean that the data lake has Raz running. You have to check if Raz is turned on for this particular data lake.

Reply:
I agree; please see `source.generalClusterConfigs.enableRangerRaz`, as done e.g. [here](https://github.com/hortonworks/cloudbreak/blob/master/template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/raz/RangerRazDatalakeConfigProvider.java#L28).